<ui version="4.0" >
 <class>PrintindexUi</class>
 <widget class="QDialog" name="PrintindexUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="2" >
    <widget class="QPushButton" name="cancelPB" >
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="okPB" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
     <property name="default" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>78</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QCheckBox" name="subindexCB" >
     <property name="toolTip" >
      <string>Check if this index should be part (e.g., a section) of the former one.</string>
     </property>
     <property name="text" >
      <string>&amp;Subindex</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="indicesLA" >
       <property name="text" >
        <string>A&amp;vailable indexes:</string>
       </property>
       <property name="buddy" >
        <cstring>indicesCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="indicesCO" >
       <property name="toolTip" >
        <string>Select the index that shall be printed at this place of the document.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>indicesCO</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>cancelPB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
